18F/identity-idp
# LG-13665: Add option to change selected email shared with partner

## 🎫 Ticket

[LG-13665](https://cm-jira.usa.gov/browse/LG-13665)

## 🛠 Summary of changes

Adds a new flow to allow a user to change their selected email shared with a partner from their Connected Accounts dashboard screen.

## 📜 Testing Plan

1. Go http://localhost:3000
2. Sign in
3. From account dashboard, click "Connected Accounts" in sidebar
4. (If you don't have any connected accounts, sign in and consent using [sample application](https://github.com/18f/identity-oidc-sinatra))
5. Observe:
   1. If you have selected a specific email to be changed (either from consent screen or from the following steps), it is shown
   2. Otherwise, you see "Email not yet selected"
7. Click "Change"
8. Change your selected email (you may need to _add_ a second email first, unless you have "Not yet selected")
9. Click "Submit"
10. Repeat Step 5

## 👀 Screenshots

![image](https://github.com/user-attachments/assets/5d7d1141-883a-4f4e-a959-be3a0edcbbeb)
